contracts/InterchainTokenService.sol

```diff
@@ -623,6 +624,29 @@ contract InterchainTokenService is
         }
     }
 
+    /**
+     * @notice Allows the owner to migrate legacy tokens that cannot be migrated automatically.
+     * @param tokenId the tokenId of the registered token.
+     * @param data the data to pass to migrate the token.
+     */
+    function migrateLegacyToken(bytes32 tokenId, bytes calldata data) external onlyOwner {
```

Comment:
So these were the legacy custom tokens:
https://interoplabs.slack.com/archives/C05F50E50MP/p1727995328548309?thread_ts=1727798397.640599&cid=C05F50E50MP

And all of these expose OZ's `grantRole` entrypoint, but it can only be called by the admin, and so wouldn't be callable by ITS (which would only have the minter role). So, I think we'll just need to coordinate with the partners to transfer the mint role and not need this method.

Reply:
removed